### PR TITLE
Improving comment phrasing

### PR DESCRIPTION
**Pull request by cdvillard**
## Description

Several comments read like run-on sentences. This clarifies the language and what each override noted from lines 34 to 52 accomplish withing a Storybook context.

### CODE REVIEW

File: docs/docs/visual-testing-with-storybook.md [hunk 1]
```diff
@@ -33,16 +33,20 @@ configure(require.context("../src", true, /\.stories\.js$/), module)
 
 // highlight-start
 // Gatsby's Link overrides:
-// Gatsby defines a global called ___loader to prevent its method calls from creating console errors you override it here
+// Gatsby defines a global variable called ___loader to prevent its method
```
Comment (wardpeet) [inline docs/docs/visual-testing-with-storybook.md]:
```suggestion
// Gatsby Link calls the `enqueue` & `hovering` methods on the global variable ___loader.
// This global object isn't set in storybook context. We override it to empty functions (no-op),
// so gatsby link doesn't throw any errors.
```

File: docs/docs/visual-testing-with-storybook.md [hunk 2]
```diff
@@ -33,16 +33,20 @@ configure(require.context("../src", true, /\.stories\.js$/), module)
...
+// calls from creating console errors. This overrides those method calls.
 global.___loader = {
   enqueue: () => {},
   hovering: () => {},
 }
 
-// Gatsby internal mocking to prevent unnecessary errors in storybook testing environment
+// Gatsby internal mocking to prevent unnecessary errors in the Storybook
```
Comment (wardpeet) [inline docs/docs/visual-testing-with-storybook.md]:
```suggestion
// __PATH_PREFIX__ is used inside gatsby-link an other various places. For storybook not to crash we need to set it as well.
```

File: docs/docs/visual-testing-with-storybook.md [hunk 3]
```diff
@@ -33,16 +33,20 @@ configure(require.context("../src", true, /\.stories\.js$/), module)
...
+// testing environment
 global.__PATH_PREFIX__ = ""
 
-// This is to utilized to override the window.___navigate method Gatsby defines and uses to report what path a Link would be taking us to if it wasn't inside a storybook
+// Gatsby defines and uses a window.___navigate method to report what path
```
Comment (wardpeet) [inline docs/docs/visual-testing-with-storybook.md]:
```suggestion
// Navigating through a gatsby app using gatsby-link or any other gatsby component will use the `___navigate` method.
// In storybook it doesn't make sense to do an actual navigate, instead we want to log an action. Checkout the [actions addon docs](https://github.com/storybookjs/storybook/tree/master/addons/actions).
```

**Comment by laurieontech** [comment]
Thanks for the PR @cdvillard! There is currently a PR in process to do a full update of this document. I'd like to close this PR for the moment but we're welcome your comments to #21378. Thanks again!

**Comment by cdvillard** [comment]
Will do, thanks for pointing this out!

**Comment by cdvillard** [comment]
Will do, thanks for pointing this out!

File: docs/docs/visual-testing-with-storybook.md (hunk 3 above)
Comment (cdvillard) [reply docs/docs/visual-testing-with-storybook.md]: These are much better than what I could do!

**Pull request closed without merging**